lib/chat_web/live/main_live/layout/message.ex

```diff
@@ -272,7 +272,7 @@ defmodule ChatWeb.MainLive.Layout.Message do
         <Layout.Card.hashed_name room={@room_card} style_spec={:room_invite} />
       </div>
 
-      <%= unless @export? or @is_mine? do %>
+      <%= unless @export? do %>
```

Comment:
I.e. I will see all the room invites i've sent to anyone (some other users) and will not be able to join that room since it is encrypted for another user

I think we should check receiver of the message